core/src/stores/sqlite.rs

```diff
@@ -33,6 +33,8 @@ impl SQLiteStore {
     pub fn new<P: AsRef<Path>>(sqlite_path: P) -> Result<Self> {
         let manager = SqliteConnectionManager::file(sqlite_path);
         let pool = Pool::new(manager)?;
+        let c = pool.get()?;
+        rusqlite::vtab::array::load_module(&c)?;
```

Comment:
@spolu this extension is needed to support `IN ()` queries with sqlite. Is this the right place to load it ?
Also, a noob rustacean, I see this thing can error, should I check if it errored and panic if it did ?

Reply:
If it takes `c` as argument I don't see any other place to do it. Error handling seems fine.